torch/export/exported_program.py

```diff
@@ -626,7 +626,7 @@ def get_shape_env(gm):
         for k, v in shape_env.var_to_range.items()
         if k not in shape_env.replacements
     }
-    for k, v in shape_env.runtime_var_to_range.items():
+    for k, v in shape_env.var_to_range.items():
         if k not in shape_env.replacements:
             range_constraints[k] = v
```

Comment:
I think after the change this loop essentially does the exactly same thing as the comprehension above.
also I made another PR today to comment this loop and landed in main branch, seems not needed.

Reply:
ok sgtm, I'll leave this alone for now